SCons/Platform/Platform.xml

```diff
@@ -121,50 +121,123 @@ else:
 <cvar name="HOST_OS">
     <summary>
 <para>
-        The name of the host operating system used to create the Environment.
-        If a platform is specified when creating the Environment, then
-        that Platform's logic will handle setting this value.
-        This value is immutable, and should not be changed by the user after
-        the Environment is initialized.
+        The name of the host operating system.
```

Comment:
In general you should not change HOST_OS, or HOST_ARCH. They should be set by SCons and immutable.

Reply:
It's not immutable. You want to say "should be considered immutable - don't  change it"?

Reply:
Yes. Should be considered... (until we make them immutable.)